src/builders/fromNodePackageJson.node.ts

```diff
@@ -117,7 +117,7 @@ export class ComponentBuilder {
       ? data.version
       : undefined
 
-    const externalReferences = this.#extRefFactory.makeExternalReferences(data)
+    const externalReferences = this.#extRefFactory.makeExternalReferences(data, buildSystem)
```

Comment:
not a responsibility of a builder class that works with `package.json`-like data